src/table/utils/handle-accessibility.ts

```diff
@@ -145,3 +145,11 @@ export const announceSelectionState = (
       : announce({ keys: ['SNTable.SelectionLabel.NotSelectedValue'] });
   }
 };
+
+export const handleKeyToCopyCell = async (cell: Cell) => {
```

Comment:
This is not really a handler. Functions that handles events. This is just a generic function  copying the value of the cell. Rename to `copyCellValue`

Reply:
also, it has nothing to do with a11y. But let's keep it here for now, not sure where it should be

Reply:
what is a11y? 😅

Reply:
accessibility